### ISSUE TITLE
Invalid HTML: closing carat (>) missing from html tag

### ISSUE THREAD

**Issue by erinchambers** [CDRH/orchid#330]
The `<html>` tag is losing its closing `>` in our HTML output. Turns out it's missing from the Orchid Rails view, `app/views/layouts/application.html.erb` - should be at the end of [Line 2](https://github.com/CDRH/orchid/blob/d0b4115e23367f2b0fc02712259683234b801e4b/app/views/layouts/application.html.erb#L2) in that file.

See [Petitioning for Freedom's homepage HTML](view-source:https://petitioningforfreedom.unl.edu/) for an example.
